tests/ops/qubit/test_state_prep.py

```diff
@@ -82,6 +82,32 @@ def test_StatePrep_decomposition(self):
         assert isinstance(ops1[0], qml.MottonenStatePreparation)
         assert isinstance(ops2[0], qml.MottonenStatePreparation)
 
+    def test_StatePrep_padding(self):
```

Comment:
follow snake case naming here and below - `test_state_prep_padding`

Reply:
Are you sure here? I'm writing StatePrep referring to the template